.npmignore

```diff
@@ -0,0 +1,9 @@
+jobs/
+node_modules/
+.gitignore
+.npmrc
+CODEOWNERS
+SECURITY.md
```

Comment:
It seems like SECURITY.md is recommended to keep?

Reply:
I'm not 100% sure though.

Reply:
![image](https://github.com/user-attachments/assets/cb92d8a7-cbc9-443e-b773-5a1a903b1939)

Reply:
No, I'm pretty sure SECURITY.md is not needed.

Reply:
ok